homeassistant/components/ambient_network/strings.json

```diff
@@ -0,0 +1,104 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Region Selection",
+        "description": "Choose the region you want to survey in order to locate Ambient personal weather stations.",
+        "data": {
+          "latitude": "[%key:common::config_flow::data::latitude%]",
+          "longitude": "[%key:common::config_flow::data::longitude%]"
+        }
+      },
+      "station": {
+        "title": "Station Selection",
+        "description": "Select the weather station you want to add to Home Assistant.",
+        "data": {
+          "station": "Station"
+        }
+      }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "no_stations_found": "Did not find any stations in the selected region."
+    }
+  },
+  "entity": {
+    "sensor": {
+      "pm25": {
+        "name": "PM2.5"
+      },
```

Comment:
Unused

(please check the other keys if they are used)

Reply:
Removed.